### PR TITLE
Fhdds 1134 variations ui

### PR DESCRIPTION
**Pull request by nubz**

### CODE REVIEW

File: app/uk/gov/hmrc/fhregistrationfrontend/views/EditMode.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.fhregistrationfrontend.views
+
+object EditMode extends Enumeration {
```
Comment (co-category) [inline app/uk/gov/hmrc/fhregistrationfrontend/views/EditMode.scala]: EditMode looks like it should be named just Mode?

File: conf/messages [hunk 2]
```diff
@@ -112,6 +112,9 @@ fh.generic.create_new_application_link=Create a new application
 
 fh.generic.check=Check your answers before sending your application
 fh.unsaved_summary=Your changes will not be submitted until you click ''Save and continue'' and complete the declaration on the next page
+fh.summary.new.unsavedChange=Your changes will not be submitted until you click ''Save and continue'' and complete the declaration on the next page
```
Comment (cosmin-marian) [inline conf/messages]: Should be uppercase New (same for next two lines)

File: app/uk/gov/hmrc/fhregistrationfrontend/views/EditMode.scala (hunk 1 above)
Comment (nubz) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/EditMode.scala]: this is something we thought about and it only relates to how users can edit the data, I can change throughout if you like?

Comment (nubz) [reply app/uk/gov/hmrc/fhregistrationfrontend/views/EditMode.scala]: I have renamed the object and params to Mode

**Pull request merged**